Formula/unum.rb

```diff
@@ -0,0 +1,17 @@
+class Unum < Formula
+  desc "Interconvert numbers, Unicode, and HTML/XHTML entities"
+  homepage "https://www.fourmilab.ch/webtools/unum/"
+  url "https://www.fourmilab.ch/webtools/unum/prior-releases/3.2/unum.tar.gz"
+  sha256 "d290070f4be54addacac7318dfb2c0bfde71690bba51f99ecf64671b71801d2a"
+
+  def install
+    system "pod2man", "unum.pl", "unum.1"
+    mv "unum.pl", "unum"
+    bin.install "unum"
```

Comment:
Just an FYI, you can rename and install with one step:
```
bin.install "unum.pl" => "unum"
```

Reply:
Fixed in 13912e503c.  Note that this syntax is not documented at https://docs.brew.sh/Formula-Cookbook.

Reply:
Thanks for pointing that out. PR for docs change has been opened at https://github.com/Homebrew/brew/pull/7600.